frontend/tests/playwright.config.ts

```diff
@@ -10,7 +10,7 @@ import { defineConfig, devices } from "@playwright/test";
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
-  timeout: 50000,
+  timeout: 75000,
```

Comment:
were you seeing more timeouts after the upgrade?

Reply:
Yes, partly because of the issue I fixed with updating one test and removing another. At this point I don't think it hurts to leave at the higher value.